src/web/src/components/BannerButtons.tsx

```diff
@@ -45,8 +54,10 @@ const LandingButtons = () => {
 
   return (
     <div
-      className={classes.buttonsContainer}
-      style={user?.isRegistered ? { width: '155px' } : { width: '250px' }}
+      className={`${classes.buttonsContainer} ${
```

Comment:
I'm not sure about this.  Does it work to use `${classes.buttonsContainer}` like this?  What does it resolve to?  It's an Object not a String, right?  Also, like 60 needs to go.

Reply:
@humphd Yes, it works; it resolves to the name of the classes that MUI create and select them:
```
console.log(
    `${classes.buttonsContainer} ${
      user?.isRegistered ? classes.userSignedInClass : classes.userNotSignedClass
    }`
  );
```

`console: makeStyles-buttonsContainer-14 makeStyles-userNotSignedClass-16`

Since we are just using multiple classes here, I think we are good to go. But if we start to see more cases like this, we can add a package to deal with multiple classes. But I found this way the best way of sync react + MUI until now.
I forgot line 60, sorry. I will delete it.